README.md

```diff
@@ -58,12 +73,31 @@ Edge is known to build with this exact tool set. Updated versions of these tools
 
     yarn start
 
+<br>
+<br>
+<br>
+<br>
+
+## Getting Started - iOS Setup
+
+### Install CocoaPods
+
+    brew install cocoapods
```

Comment:
I know you can use both brew and gem.
Is there a specific reason you changed it from gem to brew?

Reply:
Consistency. Thoughts?